examples/simple/main.go

```diff
@@ -89,7 +94,7 @@ func main() {
 		panic("-data-root must be set")
 	}
 
-	log := klogr.New()
+	log := klog.Background()
```

Comment:
```suggestion
	log := klog.TODO()
```

Nit, but according to the docs this is "more correct".